### PR TITLE
fix bad build from unlinked variable

### PR DESCRIPTION
**Pull request by lumjjb**
@pxp928 attempt to fix build from last PR

### CODE REVIEW

File: themes/hugo-fresh/layouts/partials/hero-body.html [hunk 1]
```diff
@@ -5,6 +5,8 @@
 {{- $button1Link := index $hero "button1link" }}
 {{- $button2Text := index $hero "button2text" }}
```
Comment (pxp928) [inline themes/hugo-fresh/layouts/partials/hero-body.html:6]: button2text: still also shows "Now in Beta!" which is wrong?

Comment (pxp928) [reply themes/hugo-fresh/layouts/partials/hero-body.html:6]: like the changes from the other PR did not register?

Comment (lumjjb) [reply themes/hugo-fresh/layouts/partials/hero-body.html:6]: i think the PR didnt register because the build failed :|

**Review by pxp928 — APPROVED** [review]

File: themes/hugo-fresh/layouts/partials/hero-body.html (hunk 1 above)
Comment (pxp928) [reply themes/hugo-fresh/layouts/partials/hero-body.html:6]: ah I see

**Pull request merged**